packages/amplify-schema-validator/src/index.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable global-require */
+
+import { parse } from 'graphql';
+
+const allValidators = [
+  require('./validators/index-scalar-types').validate,
+  require('./validators/relation-required-with-belongs-to').validate,
+  require('./validators/two-many-to-many-locations').validate,
+];
+
+export const validateSchema = (schemaString: string): boolean => {
+  const schema = parse(schemaString);
+  allValidators.forEach((validator) => {
+    validator(schema);
```

Comment:
n.p. but probably name this validate instead of validator. or grab the validator and execute validate.

Reply:
Also, instead of throwing on the first error, does it make sense to structure this to return a array of errors, or validation exceptions?

Reply:
I like that idea! 

Think it makes sense if each validator returns an array of error objects, and the root `validate` method concatenates all the validator responses and passes if the resulting array is empty. Otherwise it throws a new error with all the validation failure reasons in one error.